tests/unit/s2n_mem_usage_test.c

```diff
@@ -57,6 +74,27 @@ ssize_t get_vm_data_size()
     fclose(status_file);
 
     return data * page_size;
+
+#elif defined (__FreeBSD__)
+    pid_t ppid = getpid();
+    int pidinfo[4];
+    pidinfo[0] = CTL_KERN;
+    pidinfo[1] = KERN_PROC;
+    pidinfo[2] = KERN_PROC_PID;
+    pidinfo[3] = (int)ppid;
+
+    struct kinfo_proc procinfo;
+    segsz_t lsize;
```

Comment:
We've had tests break before because some very old compilers don't automatically zero-init variables, so we're pretty paranoid about initializing them. Can you zero-init procinfo and maybe move the declaration of lsize down to where you assign it so that it's clear that it's not used before being initialized?

Reply:
Never can be too paranoid about memory safety :)